### PR TITLE
LPS-129316 Cookie Banner

### PR DESCRIPTION
**Pull request by epgarcia**
Hi @liferay-echo,

This is the fragment collection for cookie banners, with a simple cookie banner. I will extend this collection with more examples, based on the feedback from LOL. 

@marcoscv-work please review the design aspects with @atjuan. 

Thanks!

### CODE REVIEW

**Comment by epgarcia** [comment]
ci:test:relevant

**Comment by epgarcia** [comment]
ci:test:sf

**Comment by epgarcia** [comment]
ci:test:sf

**Comment by epgarcia** [comment]
ci:test:relevant

**Comment by marcoscv-work** [comment]
After the meeting with Juan, Pablo and Victor to review the PR we decided to make some changes:

- Keep the background variable with gray (waiting for default value in the fragment bg)
- Remove lead (without the big text)
- Better secondary and small button (a btn-sm)
- Reduce the margin between text and button "mb-3"

**Future interesting ideas:**
- It would be a good idea in the future for it to be a composition (when it can be done)
- It would be interesting to add a horizontal version (alternative markup or alternative CSS)

I'm working on the changes!

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-cookie-banners/src/main/java/com/liferay/fragment/collection/contributor/cookie/banners/CookieBannersFragmentCollectionContributor.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	public String getFragmentCollectionKey() {
+		return "COOKIE_BANNERS";
+	}
+
+	@Override
+	public ServletContext getServletContext() {
+		return _servletContext;
+	}
+
+	@Reference(
```
Comment (jkappler) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-cookie-banners/src/main/java/com/liferay/fragment/collection/contributor/cookie/banners/CookieBannersFragmentCollectionContributor.java:46]: Hi @epgarcia, this isn't necessary if there's no widget in the fragments, so we can remove it

**Comment by epgarcia** [comment]
@jkappler Thanks for the hint, I removed it.

**Comment by epgarcia** [comment]
ci:test:relevant

**Comment by epgarcia** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcoscv-work** [comment]
Hey guys, I wasn't able to update this PR, here the commit adding the design improvements:

https://github.com/liferay-echo/liferay-portal/pull/4300/commits/3f73aa99e7fe9b2ccaea044bbb112fedfde61f6b

**Comment by ealonso** [comment]
duplicated see: https://github.com/liferay-echo/liferay-portal/pull/4300

**Comment by marcoscv-work** [comment]
please, someone to push https://github.com/liferay-echo/liferay-portal/commit/3f73aa99e7fe9b2ccaea044bbb112fedfde61f6b to this PR !

**Pull request closed without merging**